userguide/content/en/docs/Getting started/_index.md

```diff
@@ -255,6 +262,30 @@ gcs_engine_id = "011737558837375720776:fsdu1nryfng"
 
 To use your own Custom Search Engine, replace the value in the `gcs_engine_id` with the ID of your own search engine.
 
+## Known issues
+
+### MacOS
+
+#### `too many open files` - File Descriptors for "hugo server"
+
+By default, MacOS permits a very small number of open File Descriptors which can cause the following error:
```

Comment:
cause the following error when building and previewing your site locally with `hugo server`, particularly with larger sites or more recent `hugo` builds: